jaeger-core/src/test/java/com/uber/jaeger/UtilsTest.java

```diff
@@ -49,9 +49,30 @@ public void testIpToInt32EmptyIpException() {
   }
 
   @Test
-  public void testIpToInt32() {
+  public void testIpToInt32_localhost() {
     int expectedIp = (127 << 24) | 1;
     int actualIp = Utils.ipToInt("127.0.0.1");
     assertEquals(expectedIp, actualIp);
   }
+
+  @Test
+  public void testIpToInt32_above127() {
+    int expectedIp = 176750850;
```

Comment:
Makes sense. I've also changed assertions to `assertThat(x, equals(y))`, I think that they are more readable